app/controllers/songs_controller.rb

```diff
@@ -65,48 +66,52 @@ def adjust_lang_params
 
   def set_songs_to_check
     @songs_to_check ||= {}
-    @songs_to_check[:changed] = Song.recently_changed.map { |s|
-      {
-        title: [s.custom_title, s.firstline_title, s.chorus_title].reject(&:blank?).first,
-        model: s,
-        edit_timestamp: s.updated_at
-      }
-    }
+    @songs_to_check[:changed] = sort_songs(Song.recently_changed.map { |song| admin_song_entry(song.titles.values.first, song) })
+    @songs_to_check[:duplicates] = sort_songs(Song.duplicates.map { |song| admin_song_entry(song.titles.values.first, song) }) if super_admin
```

Comment:
`song.titles.values.first` could be replaced with `song.firstline_title` ? makes it a bit clearer and shorter.

Also, I think currently `@songs_to_check` are not removed from `@songs` - so any songs up the top will also appear further down in the list - not a big deal though.

Reply:
`song.titles.values.first` lets custom titles take precedence

you're right about the overlap in lists, I'll add that to the trello board